### ISSUE TITLE
Reset Delay button not shown when app is Running

### ISSUE THREAD

**Issue by pierlo-upitup** [mesosphere/marathon#1844]
The Reset Delay button is not shown when deployment is successful but tasks have failed afterwards.
The button should always be shown when the app is in the queue and the delay is not overdue, regardless of the app state.

To reproduce:

- start an application with sufficiently large number of tasks (1000)
- after a successful deployment, cause a lot of task failures
- the API endpoint (/queue) returns a delay, but button is not shown

**Comment by kolloch** [comment]
This is still happening with a build as recent as with the submodule up-to-date:

```
commit 7b5ec0a83dd7c14e9eea6cada9bb0df4b0c8a4dc (HEAD -> master, origin/master)
```

It is fairly easy to reproduce on my machine:

* Create an app with 
  * Command: "sleep 10; false"
  * CPU: 0.01
  * Instances: 50
* Let the deployment complete (if it cannot complete successfully in that time frame on you machine, use a higher sleep time)
* The tasks fail, the backoff delay gets increased
* => the app is still displayed as "running" in the UI, the delay is not shown and there is now way to reset the delay in the UI

```http
GET /v2/queue HTTP/1.1
Accept: */*
Accept-Encoding: gzip, deflate
Connection: keep-alive
Host: localhost:8080
User-Agent: HTTPie/0.9.2



HTTP/1.1 200 OK
Cache-Control: no-cache, no-store, must-revalidate
Content-Type: application/json; qs=2
Expires: 0
Pragma: no-cache
Server: Jetty(8.1.15.v20140411)
Transfer-Encoding: chunked
X-Marathon-Leader: http://dipmac:8080

{
    "queue": [
        {
            "app": {
                "acceptedResourceRoles": null, 
                "args": null, 
                "backoffFactor": 1.15, 
                "backoffSeconds": 1, 
                "cmd": "sleep 10; false", 
                "constraints": [], 
                "container": null, 
                "cpus": 0.01, 
                "dependencies": [], 
                "disk": 0.0, 
                "env": {}, 
                "executor": "", 
                "healthChecks": [], 
                "id": "/test", 
                "instances": 50, 
                "labels": {}, 
                "maxLaunchDelaySeconds": 3600, 
                "mem": 16.0, 
                "ports": [
                    10000
                ], 
                "requirePorts": false, 
                "storeUrls": [], 
                "upgradeStrategy": {
                    "maximumOverCapacity": 1.0, 
                    "minimumHealthCapacity": 1.0
                }, 
                "uris": [], 
                "user": null, 
                "version": "2015-08-12T14:17:24.142Z"
            }, 
            "count": 50, 
            "delay": {
                "overdue": false, 
                "timeLeftSeconds": 569
            }
        }
    ]
}
```

<img width="807" alt="screenshot 2015-08-12 16 25 28" src="https://cloud.githubusercontent.com/assets/339354/9226434/ef467f50-410e-11e5-9e38-cf43aa3da408.png">

**Comment by kolloch** [comment]
In this state, the UI Code only periodically hits the following endpoints:

* http://localhost:8080/v2/apps//test (also note the double `/`)
* http://localhost:8080/v2/deployments

Since it does not query the queue endpoint, it cannot know the delay.

**Comment by aldipower** [comment]
Due to submodule issues - the ui branch wasn't updated with 'git pull origin master' - you unfortunately tested a state where this feature isn't included. 
I am very sorry for that. :( Thank you a lot for taking care of describing this issue so carefully here!
This feature is included automatically in the releasing-0.10.0 branch or mv/webjar branch of Marathon.
The mv/webjar enables the webjar and discards the annoying submodule, which avoids such problems in the future.
